gef-ui/server/controllers/application-details/index.js

```diff
@@ -27,34 +24,36 @@ export const applicationDetails = async (req, res) => {
     const canSubmit = exporterStatus.code === PROGRESS.COMPLETED
       && coverStatus.code === PROGRESS.COMPLETED
       && facilitiesStatus.code === PROGRESS.COMPLETED; // All statuses are set to complete
-
+    const userAuthorisationLevels = getUserAuthorisationLevelsToApplication(user, application);
+    const previewMode = !userAuthorisationLevels.includes('EDIT');
```

Comment:
Rather than 'EDIT', should we reference to the authorisation level constant in utils/user-authorisation-level.js to ensure its always the same?

Reply:
good point ... will do

Reply:
done